### PR TITLE
[FEATURE] Prepare EAC.JS for browser TimeNode functionality.

### PR DESCRIPTION
**Pull request by e00dan**
Got TimeNode logging functionality working. This is required for the DAPP.

Requesting @kosecki123 and @lsaether input on this.

PR in DAPP: https://github.com/chronologic/eth-alarm-clock-dapp/pull/54

### CODE REVIEW

**Comment by lsaether** [comment]
On first glance it's looking good.  It looks like you were able to figure something out with Loki? If that proves to be a problem we could always use browser cache.

**Comment by e00dan** [comment]
We're using jQuery based browser adapter for Loki. After these changes package is working good so far for the dapp. I'm cleaning up the dapp PR. @lsaether

File: src/client/config.js [hunk 1]
```diff
@@ -14,10 +14,16 @@ class Config {
...
-    autostart
+    autostart,
+    logger
   ) {
     this.scanSpread = scanSpread
-    this.logger = new Logger(logfile, logLevel)
+
+    if (logger) {
+      this.logger = logger
+    } else  {
```
Comment (josipbagaric) [inline src/client/config.js]: Multiple spaces after `else`.

File: src/index.js [hunk 2]
```diff
@@ -16,10 +17,12 @@ module.exports = (web3) => {
       Scheduler,
       TxRequest,
       Util: Util(),
+      alarmClient
```
Comment (josipbagaric) [inline src/index.js]: Maybe make it start with an uppercase letter to match the rest?

Comment (e00dan) [reply src/index.js]: True that.

**Comment by e00dan** [comment]
@Bagaric fixed. @kosecki123 please review.

**Pull request merged**